modules/standard/IO.chpl

```diff
@@ -4815,7 +4815,7 @@ that ``%n`` and ``%t`` are equivalent to ``%r`` for real conversions and ``%i``
 for numeric conversions; so these are also equivalent to ``%i`` ``%d`` or
 ``%g`` in C. Also note that Chapel format strings includes many capabilities
 not available with C formatted I/O routines - including quoted strings,
-binary numbers, complex numbers, and raw binary I/O.
+binary numbers, complex numbers, and raw binary I/O *(deprecated)*.
```

Comment:
I would remove the relevant reference here instead of just marking it as deprecated, I think it might confuse people otherwise